### DISCUSSION TITLE
support vertx-core-4.x plugin

### DISCUSSION THREAD

**Discussion by alen-yao** [apache/skywalking#7794] [General]
Hi, Pls help to support vertx-core-4.x plugin, or where can I find the source code for apm-vertx-core-3.x-plugin-8.7, thx.

**Comment by wankai123** [comment]
https://github.com/apache/skywalking-java/tree/main/apm-sniffer/apm-sdk-plugin/vertx-plugins/vertx-core-3.x-plugin

**Reply by alen-yao** [reply]
thx
